test/AspNetCore.Mvc.HttpActionResults.ClientError.Test/RequestUriTooLongResultTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace AspNetCore.Mvc.HttpActionResults.ClientError.Test
+{
+    using AspNetCore.Mvc.HttpActionResults.ClientError.Test.Common;
+
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+
+    using Xunit;
+
+    public class RequestUriTooLongResultTests : BaseHttpResultTests
+    {
+        [Fact]
+        public async void RequestUriTooLongResultShouldSetStatusCodeCorrectly()
+        {
+            // Arrange
+            var fakeContext = this.CreateFakeActionContext();
+            var result = new RequestUriTooLongResult();
+
+            // Act
+            await result.ExecuteResultAsync(fakeContext);
```

Comment:
+ the guys from the AspNetCore do it in the same way for their ActionResult tests: https://github.com/aspnet/Mvc/blob/760c8f38678118734399c58c2dac981ea6e47046/test/Microsoft.AspNetCore.Mvc.WebApiCompatShimTest/InternalServerErrorResultTest.cs